help/destinations/catalog/crm/microsoft-dynamics-365.md

```diff
@@ -0,0 +1,213 @@
+
+| Credential | Description | Example |
+| --- | --- | --- |
+| Username | Your [!DNL Dynamics 365] account username | - |
+| Password | Your [!DNL Dynamics 365] account password | - |
+| Environment URL | See [[!DNL Dynamics 365] documentation](https://docs.microsoft.com/en-us/dynamics365/customerengagement/on-premises/developer/org-service/discover-url-organization-organization-service?view=op-9-1) for additional guidance. | If your domain is as below, you need the highlighted value.<br> *`org57771b33`.crm.dynamics.com* |
+
+## Supported identities {#supported-identities}
+
+[!DNL Microsoft Dynamics 365] supports update of identities described in the table below. Learn more about [identities](/help/identity-service/namespaces.md).
+
+|Target Identity|Example|Description|Considerations|
+|---|---|---|---|
+| `contactId` | `7eb682f1-ca75-e511-80d4-00155d2a68d1` | Unique identifier for a contact.| **Mandatory**, Refer to the [[!DNL Microsoft Dynamics 365] documentation](https://docs.microsoft.com/en-us/dynamics365/customerengagement/on-premises/developer/entities/contact?view=op-9-1) for further details. |
+
+## Export type and frequency {#export-type-frequency}
+
+Refer to the table below for information about the destination export type and frequency.
+
+| Item | Type | Notes |
+---------|----------|---------|
+| Export type | **[!UICONTROL Profile-based]** | <ul><li>You are exporting all members of a segment, together with the desired schema fields *(for example: email address, phone number, last name)*, according to your field mapping.</li><li> Platform segment statuses are exported to [!DNL Dynamics 365] by specifying their corresponding custom field attribute in [!DNL Dynamics 365], within the **[!UICONTROL Activate Destination]** > **[!UICONTROL Schedule segment export]** > **[!UICONTROL Mapping ID]** field.</li></ul> |
+| Export frequency | **[!UICONTROL Streaming]** | <ul><li>Streaming destinations are "always on" API-based connections. As soon as a profile is updated in Experience Platform based on segment evaluation, the connector sends the update downstream to the destination platform. Read more about [streaming destinations](/help/destinations/destination-types.md#streaming-destinations).</li></ul>|
+
+{style="table-layout:auto"}
+
+## Connect to the destination {#connect}
+
+> [!IMPORTANT]
+> 
+> To connect to the destination, you need the **[!UICONTROL Manage Destinations]** [access control permission](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+To connect to this destination, follow the steps described in the [destination configuration tutorial](../../ui/connect-destination.md). In the configure destination workflow, fill in the fields listed in the two sections below.
+
+![Catalog](../../assets/catalog/crm/microsoft-dynamics-365/catalog.png)
```

Comment:
There is no need for this screenshot, as this process is already illustrated in the destination config tutorial.

Reply:
Screenshot has been removed.